docs/01_nodeos/02_usage/03_development-environment/10_local-single-node-testnet-consensus.md

```diff
@@ -0,0 +1,202 @@
+---
+content_title: Local Single-Node Testnet With Consensus Protocol
+link_text: Local Single-Node Testnet With Consensus Protocol
+---
+
+## Goal
+
+This section describes how to set up a single-node blockchain configuration running on a single host with consensus protocol enabled.  This is referred to as a _**single host, single-node testnet with consensus**_.  We will set up one node on your local computer and have it produce blocks. The following diagram depicts the desired single host testnet.
```

Comment:
Suggestion
Can hyperlink the text **consensus protocol** to link to:
https://developers.eos.io/welcome/latest/protocol/consensus_protocol